assets/js/components/dashboard-splash/dashboard-splash-app.js

```diff
@@ -22,6 +22,12 @@
 import DashboardSplashMain from 'GoogleComponents/dashboard-splash/dashboard-splash-main';
 import { Suspense as ReactSuspense, lazy as ReactLazy } from 'react';
 
+/**
+ * WordPress dependencies
+ */
+import { Component, Fragment, Suspense as WPSuspense, lazy as WPlazy } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
```

Comment:
I'm not sure that ever got merged; at any rate it was weird because even the version of React we load depends on what Gutenberg/other scripts expose. I'll have a look though. 👍

Reply:
This still needs to be revisited, it should import from `util/react-features`.